kbase-extension/static/kbase/templates/data_staging/file_path.html

```diff
@@ -1,4 +1,12 @@
-<button id="refresh" class="btn btn-xs btn-default">
+<span class="kb-data-stagingarea">Staging Area </span>
```

Comment:
Is the space needed after the "Staging Area" text? If so, it might be best to add some margin to the class.